src/store/tools/paint.ts

```diff
@@ -80,18 +80,6 @@ export const usePaintToolStore = defineStore('paint', () => {
    * @param segValue
    */
   function setActiveSegment(this: _This, segValue: Maybe<number>) {
-    if (segValue) {
-      if (!activeSegmentGroupID.value)
-        throw new Error('Cannot set active segment without a labelmap');
-
-      const segmentGroupStore = useSegmentGroupStore();
-      const { segments } =
-        segmentGroupStore.metadataByID[activeSegmentGroupID.value];
-
-      if (!(segValue in segments.byValue))
-        throw new Error('Segment is not available for the active labelmap');
-    }
-
```

Comment:
Can you explain how this block of code affected the paint tool not loading properly through a session file?

Reply:
Yes.  `usePaintStore.deserialize` calls `setActiveSegment` before `activeSegmentGroupID` is set.  Later, activeSegmentGroupID will be set by `useToolStore.setCurrentTool` -> `usePaintStore.activateTool` -> `setActiveLabelmapFromImage`

Reply:
I think then that we can solve this by calling `setActiveLabelmap(segmentGroupIDMap[paint.activeSegmentGroupID])` before `setActiveSegment` in `deserialize`. I think keeping the code in `setActiveSegment` is good to ensure we are getting a valid segment value. Thoughts?

Reply:
Would be nice to keep the checks.   I rebased a fix.  Turns out segmentIDMap was not getting passed down to `paint.deserialize`.  Also, can't `setActiveSegment` without a segmentGroup, so only calling `setActiveSegment` if one is deserialized.